styles/notifications-log.less

```diff
@@ -24,6 +24,10 @@
       opacity: .5;
       background: @base-background-color;
     }
+
+    .notifications-clear-log {
+      float: right;
+    }
```

Comment:
@UziTech: When you have a moment, can you share screenshots of this new icon is rendered in each of the UI themes that ship with Atom? I suspect it renders just fine, but we want to make sure. 😅

Reply:
### Atom Dark
![image](https://user-images.githubusercontent.com/97994/62493383-912af600-b796-11e9-8635-9ba900db8c83.png)

### Atom Light
![image](https://user-images.githubusercontent.com/97994/62493405-9daf4e80-b796-11e9-98bb-3404c2d4f9f0.png)

### One Dark
![image](https://user-images.githubusercontent.com/97994/62493413-a43dc600-b796-11e9-87c1-363ce0403616.png)

### One Light
![image](https://user-images.githubusercontent.com/97994/62493417-a99b1080-b796-11e9-8d4b-4df1505c76ac.png)